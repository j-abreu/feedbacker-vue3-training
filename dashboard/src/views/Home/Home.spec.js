import Home from '.'
import { routes } from '../../router'
import { createRouter, createWebHistory } from 'vue-router'
import { shallowMount } from '@vue/test-utils'

const router = createRouter({
  history: createWebHistory('/'),
  routes
})

describe('<Home />', () => {
  it('should be render home', async () => {
    router.push('/')
    await router.isReady()
    const wrapper = shallowMount(Home, {
      global: {
        plugins: [router]
      }
    })

    expect(wrapper.html()).toMatchSnapshot()
  })

})