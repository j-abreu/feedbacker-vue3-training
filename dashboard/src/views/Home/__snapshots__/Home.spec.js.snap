// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should be render home 1`] = `
<custom-header-stub></custom-header-stub>
<contact-stub></contact-stub>
<div class="flex justify-content py-10 bg-brand-gray">
  <p class="font-medium text-center text-gray-800">Feedbacker Â© 2022</p>
</div>
`;
